packages/serverless-contracts/src/contracts/apiGateway/__tests__/httpApiContract.test.ts

```diff
@@ -1,5 +1,12 @@
 /* eslint-disable max-lines */
 import { ApiGatewayContract } from '../apiGatewayContract';
+import {
+  getCompleteTrigger,
+  getFullContractSchema,
+  getInputSchema,
+  getOpenApiDocumentation,
+  getTrigger,
+} from '../features';
```

Comment:
Maybe we could benefit from the `features` split to also test each feature independently in separate test files